pages.es/windows/cpush.md

```diff
@@ -0,0 +1,8 @@
+# cpush
+
+> Este comando es un alias de `choco push`.
+> Más información: <https://docs.chocolatey.org/en-us/create/commands/push>.
+
+- Ver documentación para el comando original:
+
+`tldr choco push`
```

Comment:
I think we should recommend the page with `kebab-case`, both should work in the tldr client.
```suggestion
`tldr choco-push`
```